classes/Http/Action/Profile/EditAction.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Http\Action\Profile;
+
+use OpenCFP\Domain\Model;
+use OpenCFP\Domain\Services;
+use OpenCFP\PathInterface;
+use Symfony\Component\HttpFoundation;
+use Symfony\Component\Routing;
+use Twig_Environment;
+
+final class EditAction
+{
+    /**
+     * @var Services\Authentication
+     */
+    private $authentication;
+
+    /**
+     * @var PathInterface
+     */
+    private $path;
+
+    /**
+     * @var Twig_Environment
+     */
+    private $twig;
+
+    /**
+     * @var Routing\Generator\UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    public function __construct(
+        Services\Authentication $authentication,
+        PathInterface $path,
+        Twig_Environment $twig,
+        Routing\Generator\UrlGeneratorInterface $urlGenerator
+    ) {
+        $this->authentication = $authentication;
+        $this->path           = $path;
```

Comment:
Since the information is static anyway, why don't we pass `$path->downloadFromPath()` to this class instead of the whole path object?